git_theta/lsh/base.py

```diff
@@ -0,0 +1,29 @@
+"""Base class for computing locality-sensitive hashes"""
+
+import abc
+from git_theta.lsh import get_randomness_pool
+from git_theta.lsh.types import Signature, Parameter
+
+
+class HashFamily(metaclass=abc.ABCMeta):
+    def __init__(self, signature_size: int):
```

Comment:
Talked to @craffel today and he pointed out that if you create a `np.random.BitGenerator` object by picking one of the classes that inherit from `BitGenerator` then you can get consistent randomness across systems. Instead of pre-computing and distributing `.npy` files, the random pool is initialized from a Mersenne Twister PRNG seeded with a fixed value. See `__init__` in `RandomnessPool`.